alpha/property/property.go

```diff
@@ -257,6 +257,9 @@ func MustBuildGVKRequired(group, version, kind string) Property {
 func MustBuildBundleObject(data []byte) Property {
 	return MustBuild(&BundleObject{Data: data})
 }
+func MustBuildBundleObjectData(data []byte) Property {
+	return MustBuild(&BundleObject{Data: data})
+}
```

Comment:
This looks identical to `MustBuildBundleObject` above. Do we need both?

Reply:
Vestigial.  Cleaned up.